src/utils/organize/people.spec.ts

```diff
@@ -0,0 +1,86 @@
+import isEqual from 'lodash.isequal';
+import { nestByParentId } from './organizations';
+import { ZetkinOrganization } from 'types/zetkin';
+import {
+  getConnectedOrganisations,
+  getPersonOrganisations,
+  PersonOrganisation,
+} from './people';
```

Comment:
Great use of tests!

My only critique is that this module contains tests for two other modules (`./organizations` and `./people`), but is named as if it contains tests only for the `people` module. I think these should be restructured to avoid such confusion, either by merging the two modules or splitting the tests.